packages/react-tinacms/src/use-form.ts

```diff
@@ -104,7 +104,7 @@ function updateFormLabel(label?: string, form?: Form) {
  */
 function updateFormValues(values: any = {}, form?: Form) {
   React.useEffect(() => {
-    if (!form || typeof values === 'undefined') return
+    if (!form || Object.keys(values).length == 0) return
```

Comment:
This sort of changes the semantics.

Did you see if removing the `= {}` from the arguments worked?

i.e.

```ts
function updateFormValues(values: any, form?: Form) {
```

Reply:
yeah I guess that makes much more sense, updated